src/mpid/ch3/src/ch3u_win_fns.c

```diff
@@ -7,9 +7,56 @@
 #include "mpidimpl.h"
 #include "mpir_info.h"
 #include "mpidrma.h"
+#include "mpir_memkind.h"
```

Comment:
Are we adding this support for `ch3` too, or just `ch4`?

Reply:
CH4 was not supporting fastboxes at the time I started this. I will add it now